requirements.txt

```diff
@@ -0,0 +1,61 @@
+Babel==2.3.4
```

Comment:
Can you try removing the version numbers from the requirements.txt file? Maybe that's enough for *changes* ...